tests/api/v1/profiles/post.js

```diff
@@ -45,11 +55,36 @@ describe(`api: POST ${path}`, () => {
       });
     });
 
+    it('fail, not an admin profile', (done) => {
+      api.post(`${path}`)
+      .set('Authorization', token)
+      .send(p0)
+      .expect(constants.httpStatus.FORBIDDEN)
+      .end((err, res) => {
+        if (err) {
+          done(err);
+        }
+        expect(res.body.errors[ZERO].type).to.equal('ForbiddenError');
+        done();
+      });
+    });
+
+    it('fail, no token provided', (done) => {
+      api.post(`${path}`)
+      .send(p0)
+      .expect(constants.httpStatus.FORBIDDEN)
+      .end((err, res) => {
+        if (err) {
+          done(err);
```

Comment:
You don't have to change it in this PR but we ideally should be adding a "return" in front of done(err). If you don't add it, all the errors will fail, saying that "done" was called twice.